06_distributed_advanced.ipynb

```diff
@@ -197,6 +200,16 @@
     "collapsed": false
    },
    "outputs": [],
+   "source": [
+    "from prep import accounts_json\n",
+    "accounts_json(3, 100000, 500)"
```

Comment:
In prep.py it looks like we use `accounts_json(50, 100000, 500)`.  I don't know which option is better.  Thoughts?

Reply:
Sorry I did not read the README.md carefully enough and did not spot this through naive grepping.